connectors/katalog/pkg/connector/connector_test.go

```diff
@@ -277,7 +293,6 @@ func TestCreateAssetWthNoDestinationAssetID(t *testing.T) {
 
 		assetName := response.AssetID
 		namespace := "fybrik-system"
-		g.Expect(assetName).Should(HavePrefix(FybrikAssetPrefix))
```

Comment:
Either return this check or simplify the connector code not to use the prefix in the first place.

Reply:
I don't understand this test, let's discuss it.